### PR TITLE
add StakingCabin node to vega mainnet config

### PR DESCRIPTION
**Pull request by richard-stakingcabin**

### CODE REVIEW

File: mainnet1/mainnet1.toml [hunk 1]
```diff
@@ -44,7 +44,7 @@ Value = "mainnet"
       # "https://vega.greenfield.xyz",
       # !! not passing standard validations
       # "https://vega.xprv.io/datanode",
-      # "https://vega.mainnet.stakingcabin.com",
+      "https://vega.mainnet.stakingcabin.com",
```
Comment (fkondej) [inline mainnet1/mainnet1.toml:47]: This one doesn't work for me, but `https://vega.mainnet.stakingcabin.com:3008` works.

File: mainnet1/mainnet1.toml [hunk 2]
```diff
@@ -63,7 +63,7 @@ Value = "mainnet"
       # "https://vega.greenfield.xyz/graphql",
       # !! not passing standard validations
       # "https://vega.xprv.io/datanode/query",
-      # "https://vega.mainnet.stakingcabin.com/query",
+      "https://vega.mainnet.stakingcabin.com/query",
```
Comment (fkondej) [inline mainnet1/mainnet1.toml:66]: This one doesn't work for me, but `https://vega.mainnet.stakingcabin.com:3008/query` works.

**Review by fkondej — CHANGES_REQUESTED** [review]
Can you verify endpoints?

File: mainnet1/mainnet1.toml (hunk 2 above)
Comment (richard-stakingcabin) [reply mainnet1/mainnet1.toml]: changed, thanks

File: mainnet1/mainnet1.toml (hunk 1 above)
Comment (richard-stakingcabin) [reply mainnet1/mainnet1.toml]: changed

**Review by fkondej — APPROVED** [review]
I have confirmed the node is healthy with two tools: [vega-monitoring](https://github.com/vegaprotocol/vega-monitoring) and [Vaguer](https://github.com/vegaprotocol/vaguer).

**Pull request merged**